ShareClient/ShareGlucose+GlucoseKit.swift

```diff
@@ -41,6 +45,12 @@ extension ShareGlucose: GlucoseDisplayable {
     }
 }
 
+extension ShareGlucose {
+    public var condition: GlucoseCondition? {
```

Comment:
Wouldn't we return high/low for 401/39 here?

Reply:
Well, we can, but I wasn't sure if the 39/401 limit were the same for all versions of Dexcom. I figured that DIY folks could make that determination. However, if you are telling me that all versions of Dexcom use 39 and 401 as LOW and HIGH indicators and we can do the same thing as in Tidepool Loop then I will update this code accordingly.

Reply:
@ps2 What would you like me to do here?